VimeoNetworking/Sources/Models/VIMVideoPreference.m

```diff
@@ -38,7 +38,7 @@ - (void)didFinishMapping
     if ([self.privacy isKindOfClass:[NSString class]])
     {
         VIMPrivacy *privacy = [VIMPrivacy new];
-        privacy.view = self.privacy;
+        privacy.view = (NSString *)self.privacy;
```

Comment:
@ghking Casting here to silence a warning about `self.privacy` not being an NSString. 

🌀😱🌀 Also, why tf is `view` an `NSString` 🌀😱🌀